documentation-website/Writerside/snippets/get-started-with-exposed/app/src/main/kotlin/org/example/App.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * This source file was generated by the Gradle 'init' task
+ */
+package org.example
+
+import Tasks
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.transactions.transaction
+
+fun main() {
+    Database.connect("jdbc:h2:mem:test", driver = "org.h2.Driver")
```

Comment:
The following comment is most likely outside the scope of this tutorial, since it's a single transaction app. So feel free to ignore, I'm just mentioning this since users can download this sample and potentially try to take it further.

If a second transaction is opened after the first (to play more with the remaining task for example), the user may be surprised that the table and its data has been lost even though the app hasn't stopped. This is normal H2 behavior between connections/transactions (better explained [here](https://www.h2database.com/html/features.html#in_memory_databases)) and the solution is to use the url `jdbc:h2:mem:test;DB_CLOSE_DELAY=-1`.

I don't see this as a large issue, I'm just putting it out there.

Reply:
Great! I think this could be useful, so I added is as a side note before the "Next steps" section.